src/platform/install_osx.cpp

```diff
@@ -82,3 +84,27 @@ void osx::moveToAppFolder()
         }
     }
 }
+
+void osx::migrateProfiles()
```

Comment:
Also, you could leave a comment saying that this is temporary migration code, so that when it gets forgotten about 10 years later we'll know it can be safely removed :)